internal/storager/memdb/storage.go

```diff
@@ -0,0 +1,76 @@
+package storage
+
+import (
+	"context"
+	"net/url"
+	"sync"
+
+	appErr "github.com/rasha108bik/tiny_url/internal/errors"
+)
+
+type memDB struct {
+	locations map[string]string
+	mtx       sync.RWMutex
+}
+
+func NewMemDB() *memDB {
+	return &memDB{
+		locations: make(map[string]string),
+	}
+}
+
+func (m *memDB) StoreURL(ctx context.Context, originalURL string, shortURL string) error {
```

Comment:
```suggestion
func (m *memDB) StoreURL(_ context.Context, originalURL string, shortURL string) error {
```
что бы линтеры не ругались

Reply:
done